web/app/Services/MJML/MJML.php

```diff
@@ -40,7 +48,7 @@ public static function render(string $template_path, array $data = []): HtmlStri
         // command has a 5 second timeout
         // if rendering takes longer than that, something is very wrong
         // average render time should be in milliseconds
-        $proc = Process::fromShellCommandline('mrml render', null, null, $raw_mjml, 5);
+        $proc = Process::fromShellCommandline('mrml render', null, null, $mjml, 5);
```

Comment:
As a security note, I don't like shell invocation if direct invocation is possible. Looking at the [Symfony `Process` docs](https://symfony.com/doc/current/components/process.html), it looks like what we want is something like:
```php
$proc = new Process(['mrml', 'render']);
$proc->setTimeout(5);
$proc->run();
```

Reply:
To be clear, the way this is being executed is via `stdin` and not pasting anything into the command line. I agree not using the shell would be preferable, but how do you pass `stdin` to the `Process` instance?

Reply:
Whoops sorry, forgot to include that. See the edited code snippet above.

Reply:
Oh that's simple, okay, I'll do that.